### PR TITLE
Upload Game Dragon Ball Fusions

### PR DESCRIPTION
**Pull request by krikic**
Upload Dragon Ball Fusions with images and game.dat info.Work at 100% without workarounds

### CODE REVIEW

**Comment by Leo626** [comment]
- That is not the box art for the game.
- Those screenshots look like they aren't cropped properly. Use ShareX app to take a capture of the window region in Citra.
- Why does the title id in `testcases`  not match the one in `releases`. Would be nice to add usa and jpn region as well.
- Create the wiki entry while you're at it.

File: Dragon-Ball-Fusions/game.dat [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+github_issues = [26]
+needs_system_files = false
+needs_shared_font = false
+
+[[ releases ]]
+title = "00040000001aaa00"
+region = "eur"
+release_date = "2017-02-17"
+
+
```
Comment (Leo626) [inline Dragon-Ball-Fusions/game.dat]: Again unnecessary new line.

**Comment by j-selby** [comment]
Your boxart's resolution is still incorrect - it should be 320x300.

Are your screenshots correct? they look pretty low quality, and some of the layouts in them are wrong.

File: Dragon-Ball-Fusions/game.dat [hunk 2]
```diff
@@ -0,0 +1,30 @@
+title = "Dragon Ball Fusions"
+description = "Dragon Ball Fusions (ドラゴンボールフュージョンズ) is a role-playing based in Dragon Ball video game for the Nintendo 3DS."
```
Comment (j-selby) [inline Dragon-Ball-Fusions/game.dat]: Description still makes no grammatical sense.

**Comment by j-selby** [comment]
Yeah, that combat screenshot looks cropped and stretched - not great.

You are obviously using some kind of wacky screenshot tool which introduces compression. Those images look very pixelated to my eye.

File: Dragon-Ball-Fusions/game.dat [hunk 3]
```diff
@@ -0,0 +1,30 @@
+title = "Dragon Ball Fusions"
+description = "Dragon Ball Fusions is a platform-fighter videogame, developed by Ganbarion.co and distributed by Bandai Namco, based on the Dragon Ball series. The game was launched on August 4, 2016 in Japan, November 22, 2016 in America and February 17, 2017 in Europe."
```
Comment (Leo626) [inline Dragon-Ball-Fusions/game.dat]: I feel the second sentence isn't needed. The release dates are already mentioned in `releases`.

File: Dragon-Ball-Fusions/game.dat [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+github_issues = [26]
```
Comment (Leo626) [inline Dragon-Ball-Fusions/game.dat]: This leads to [this](https://github.com/citra-emu/citra/pull/26). What does this have to do with the game.

Comment (krikic) [inline Dragon-Ball-Fusions/game.dat]: Are the characteristics and the issues that i have done

Comment (j-selby) [inline Dragon-Ball-Fusions/game.dat]: That made no sense. We are asking what issue 26 has to do with the functioning of this game.

**Comment by krikic** [comment]
Now i need to improve quality of images screenshoots

**Comment by j-selby** [comment]
Next time, please don't post in *OTHER* PRs, nor use PMs on the forums for communication. Thankyou.

File: Dragon-Ball-Fusions/game.dat [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+release_date = "2016-11-22"
+
+[[ releases ]]
+title = "0004000000196D00"
+region = "jpn"
+release_date = "2016-08-04"
+
+[[ testcases ]]
+title = "0004000000196D00"
+compatibility = "1"
```
Comment (j-selby) [inline Dragon-Ball-Fusions/game.dat]: You said the game runs perfectly, yet you have a non-perfect score here. Reason?

**Comment by krikic** [comment]
Ok

2017-06-19 13:30 GMT+02:00 James <notifications@github.com>:

**Comment by krikic** [comment]
I pull the boxart with good quality

2017-06-19 13:51 GMT+02:00 James <notifications@github.com>:

File: Dragon-Ball-Fusions/game.dat (hunk 5 above)
Comment (j-selby) [inline Dragon-Ball-Fusions/game.dat]: Still need to fix this up, sorry.

**Comment by chris062689** [comment]
@j-selby @Leo626 Are these changes ready to go live?

**Comment by Leo626** [comment]
Still no wiki entry :/. Other than the compatibility rating LGTM.
